foreman.te

```diff
@@ -171,189 +174,134 @@ require{
     type cockpit_session_exec_t;
 }
 
-#######################################
+######################
 #
-# Passanger/httpd local policy
+# Ruby on Rails policy
 #
 
-# https://bugzilla.redhat.com/show_bug.cgi?id=1716944
-allow passenger_t self:capability dac_override;
-allow passenger_t self:capability sys_resource;
-allow passenger_t self:process signull;
-allow passenger_t self:tcp_socket listen;
+# Service transition
+init_daemon_domain(foreman_rails_t, foreman_rails_exec_t)
 
-# rubygem FFI is used in foreman-tasks (BZ#1670109 / RM#26951)
-allow passenger_t self:process execmem;
+# Generic domain rules
+auth_read_passwd(foreman_rails_t)
+dev_list_sysfs(foreman_rails_t)
+dev_read_sysfs(foreman_rails_t)
+hostname_exec(foreman_rails_t)
+ipa_filetrans_named_content(foreman_rails_t)
+kernel_read_network_state(foreman_rails_t)
+kernel_read_system_state(foreman_rails_t)
+sysnet_read_config(foreman_rails_t)
 
-miscfiles_read_localization(passenger_t)
+# Certificates (Foreman uses puppet certificates as well)
+miscfiles_read_certs(foreman_rails_t)
+puppet_read_config(foreman_rails_t)
 
-# Allow Foreman to connect to Foreman Proxy on port 9090 (Katello)
-# or to connect to remote Cockpit instance (via Remote Execution)
-allow passenger_t websm_port_t:tcp_socket name_connect;
+# It is not possible to do transition into SCL application
+# without calling a shell: https://github.com/sclorg/scl-utils/issues/31
+# These rules are for: /usr/libexec/foreman/tfm-ruby-selinux
```

Comment:
This particular file doesn't exist so that's a bit confusing.

Reply:
A leftover, removed.